examples/machine_learning/CMakeLists.txt

```diff
@@ -0,0 +1,109 @@
+cmake_minimum_required(VERSION 3.0)
+cmake_policy(VERSION 3.5)
+project(ArrayFire-Example-Linear-Algebra
```

Comment:
@umar456 I believe this will create a solution for each sub-category in Visual Studio with examples from that category as projects inside that solution. Is that how you intend to organize this ?